src/libprojplugin/libprojplugin.h

```diff
@@ -7,6 +7,7 @@
 #include <QVector>
 #include "libprojprojectnodes.h"
 #include "../../tools/json11/json11.hpp"
```

Comment:
#bug there musnt' be relative paths which are upper than source tree
hack on include path manipulation in build system